packages/serverpod_lints/pubspec.yaml

```diff
@@ -10,5 +10,5 @@ environment:
   sdk: '>=3.1.0 <4.0.0'
 
 dependencies:
-  flutter_lints: ^3.0.0
-  lints: ^3.0.0
+  flutter_lints: '>=3.0.0 <5.0.0'
+  lints: '>=3.0.0 <5.0.0'
```

Comment:
Suggestion: move to dev_dependencies

Reply:
This is the lints package so it depends on the actuall lints.